torchtext/experimental/checksums_fast_text.json

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
Do fasttext and glove checksums need to be in two separate files? Should we have an asset folder to save it?

Reply:
Added the scripts to compute checksums to `experimental/asset` folder